src/expr/skolem_manager.cpp

```diff
@@ -43,8 +43,9 @@ SkolemManager::SkolemManager() : d_skolemCounter(0) {}
 
 Node SkolemManager::mkPurifySkolem(Node t)
 {
+  SkolemManager* skm = t.getNodeManager()->getSkolemManager();
   // We do not recursively compute the original form of t here
-  Node k = mkSkolemFunction(SkolemId::PURIFY, {t});
+  Node k = skm->mkSkolemFunction(SkolemId::PURIFY, {t});
```

Comment:
You can make this a static call, right?

Reply:
I don't think so. The method `mkSkolemFunction` has the following declaration:
```
Node mkSkolemFunction(SkolemId id, Node cacheVal = Node::null());
```
This means that `cacheVal` is expected to be `null` in some cases, so we cannot always retrieve the node manager from it.

Reply:
You can update this to `SkolemManager::mkPurifySkolem`, which can be made static, can you do this on this PR?

Reply:
Sorry, I don't understand your request. The piece of code we are discussing is the definition of `SkolemManager::mkPurifySkolem`. This change (together with this [one](https://github.com/cvc5/cvc5/pull/11423/files#diff-c858376640a93c37c9deaf5644f0451aee7c904fbe5dd798d6eb00022a47111eR99)) makes `SkolemManager::mkPurifySkolem` static.